test/e2e/specs/site-editor/style-book.spec.js

```diff
@@ -117,13 +117,30 @@ test.describe( 'Style Book', () => {
 		).toBeVisible();
 	} );
 
-	test( 'should disappear when closed', async ( { page } ) => {
+	test( 'should disappear when closed via click event or Escape key', async ( {
+		page,
+	} ) => {
+		// Close Style Book via click event.
 		await page.click(
 			'role=region[name="Style Book"i] >> role=button[name="Close Style Book"i]'
 		);
+		await expect(
+			page.locator( 'role=region[name="Style Book"i]' ),
```

Comment:
Nit: We can hoist this locator and reuse it in the test:

```js
const styleBook = page.getByRole( 'region', { name: 'Style Book' } );

await expect( styleBook ).toBeVisible();
await styleBook.getByRole( 'button', { name: 'Close Style Book' } ).click();
```

Also it's now recommended to use `getByRole` rather than role-selector after a recent change of Playwright's doc 😅 .

Reply:
Oh, thanks for the tip! In https://github.com/WordPress/gutenberg/pull/48151/commits/d302f487b549500bc6660b331d25dd970e7ab86d I've named the const `styleBookRegion` to distinguish between it and the `styleBook` instance of the `StyleBook` class that's available to each of the tests.

I've left the other tests untouched, though I see they could be switched to `page.getByRole` in a follow-up. Happy to keep tweaking this test, though!

Reply:
This is perfect! Thanks for continue following the best practices 💯 .

FWIW, I'm also trying to build a codemod to automate the transformation from role-selector to `getByRole`, so no need to manually refactor the existing ones just yet. 👍